resources/qml/pages/ConversationsPage.qml

```diff
@@ -75,13 +75,13 @@ Page {
                 }
                 Label {
                     id: jidId;
-                    text: jid;
+                    text: qsTr("jid");
                     color: Theme.secondaryColor;
                     font.pixelSize: Theme.fontSizeTiny;
                 }
                 Label {
                     id: statusId;
-                    text: lastmessage;
+                    text: qsTr("last message");
```

Comment:
This is an id which gets replaced by c++ code with the last message of that contact. Not translateable.